src/Compilers/CSharp/Portable/Symbols/Source/SourceSimpleParameterSymbol.cs

```diff
@@ -9,9 +9,42 @@ namespace Microsoft.CodeAnalysis.CSharp.Symbols
     /// A source parameter that has no default value, no attributes,
     /// and is not params.
     /// </summary>
-    internal sealed class SourceSimpleParameterSymbol : SourceParameterSymbol
+    internal sealed class SourceSimpleParameterSymbol : SourceSimpleParameterSymbolBase
     {
         public SourceSimpleParameterSymbol(
+                  Symbol owner,
+                  TypeWithAnnotations parameterType,
+                  int ordinal,
+                  RefKind refKind,
+                  string name,
+                  ImmutableArray<Location> locations)
+                  : base(owner, parameterType, ordinal, refKind, name, locations)
+        {
+        }
+    }
+
+    internal sealed class DiscardParameterSymbol : SourceSimpleParameterSymbolBase, IDiscardSymbol
```

Comment:
ohhhhh.   That's interesting so a symbol can be *two* dfferent types of ISymbols?  i.e. an IDiscardSymbol and an IParameterSymbol?

Not sure how i feel about that :D